src/docs/releases/3.0.0.md

```diff
@@ -48,3 +48,9 @@ The following obsolete settings were removed from `LoginSettings` class
 #### Login View Update
 
 The `ExternalLogin` action has been removed from the `Account` controller. If you are using a custom `Login.cshtml` view or `Login` template, please update the external login form action. As of this update, the `ExternalLogin` action has been relocated to the `ExternalAuthentications` controller.
+
+### Sealing Types
+
+Many type commonly used by modules can be `sealed`, which improves runtime performance. While it's not mandatory, we recommend that you consider applying this improvement to your own extensions as well. We've implemented this enhancement in the following pull-request:
```

Comment:
```suggestion
Many type commonly used classes can be `sealed`, which improves runtime performance. While it's not mandatory, we recommend that you consider applying this improvement to your own extensions as well. We've implemented this enhancement in the following pull-requests:
```

Reply:
It's a one PR not many. FYI I just used the same phrases from `2.x`

Reply:
yes. but you listed the PR as bullet points implying multiple related PR. Either we remove the bullets or change it to the suggestion.